metacat-connector-polaris/src/main/java/com/netflix/metacat/connector/polaris/PolarisConnector.java

```diff
@@ -0,0 +1,25 @@
+package com.netflix.metacat.connector.polaris;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.transaction.annotation.Transactional;
+
+/**
+ * This class exposes APIs for CRUD operations.
+ */
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class PolarisConnector {
```

Comment:
We might break this into multiple services such as DatabaseService, TableService, and so on. This is pending some design decisions to be made about how much we align with vs break away from the current patterns that each connector follows.

Reply:
lets leave it as is for now and we can iterate over it.